### PR TITLE
Document how to find NumPy with CMake on cross-compiled builds

### PR DESCRIPTION
**Pull request by traversaro**
PR Checklist:

- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [x] put any other relevant information below

Proposal to fix https://github.com/conda-forge/conda-forge.github.io/issues/2260 .

### CODE REVIEW

**Comment by traversaro** [comment]
pre-commit.ci autofix

File: docs/maintainer/knowledge_base.md [hunk 1]
```diff
@@ -559,6 +559,21 @@ fi
 There are more variations of this approach in the wild. So this is not meant to be exhaustive,
 but merely to provide a starting point with some guidelines. Please look at [other recipes for more examples](https://github.com/search?q=org%3Aconda-forge+path%3Arecipe%2Fmeta.yaml+%22%5Bbuild_platform+%21%3D+target_platform%5D%22&type=code).
 
+#### Finding NumPy in cross-compiled Python packages using CMake
+
+If you are building a Python extension via CMake with NumPy and you want it work in cross-compilation, you need to preprend to the CMake invocation in your build script the following lines:
```
Comment (jaimergp) [inline docs/maintainer/knowledge_base.md:564]:
```suggestion
If you are building a Python extension via CMake with NumPy and you want it work in cross-compilation, you need to prepend to the CMake invocation in your build script the following lines:
```

**Comment by traversaro** [comment]
Sorry @jaimergp, I just realized I never asked again the review after I address your comment.

**Review by jaimergp — APPROVED** [review]
LGTM but I don't have the background to assess whether this is the right solution or not. Judging by the discussion in #2260 it looks like the best effort so far, so I'm inclined to merge regardless. I'll leave it open for a couple days before doing so in case someone from @conda-forge/core wants to chime in.

**Comment by traversaro** [comment]
> LGTM but I don't have the background to assess whether this is the right solution or not.

That is correct. My feeling is that probably there should be something that may be fixed in cmake and/or conda-forge cross-compilation setuptools to avoid the need to manually set this include directories, but I was not able to identify this.

**Comment by hmaarrfk** [comment]
Let’s merge and iterate

**Pull request merged**